### PR TITLE
Translated node-interface.md

### PR DESCRIPTION
**Pull request by GeekBoySupreme**
Hey @nutboltu, @shakib609 this is done. Do review.
Thanks

### CODE REVIEW

**Comment by GeekBoySupreme** [comment]
@nutboltu @shakib609 a soft reminder here

File: docs/docs/node-interface.md [hunk 1]
```diff
@@ -27,66 +27,66 @@ internal: {
...
 
 ### `owner`
 
-The plugin which created this node. This field is added by gatsby itself (and not the plugin).
+এই নোডটি তৈরি করে এমন প্লাগইন. এই ক্ষেত্রটি Gatsbyর দ্বারাই যুক্ত হয় (এবং প্লাগইন দ্বারা নয়)|
 
 ### `fieldOwners`
 
-Stores which plugins created which fields. This field is added by gatsby itself (and not the plugin).
+কোন প্লাগইনগুলি কোন ক্ষেত্র তৈরি করেছে তা স্টোর করে. এই ক্ষেত্রটি Gatsbyর দ্বারাই যুক্ত হয় (এবং প্লাগইন দ্বারা নয়)|
```
Comment (nutboltu) [inline docs/docs/node-interface.md]: not a blocker. we can replace full stop with "।"

File: docs/docs/node-interface.md [hunk 2]
```diff
@@ -27,66 +27,66 @@ internal: {
...
-Optional field exposing the raw content for this node that transformer plugins can take and further process.
+অপ্শনাল ক্ষেত্র যা ট্রান্সফর্মার প্লাগইনগুলি নিতে এবং আরও প্রক্রিয়া করতে পারে এমন নোডের raw কনটেন্ট উন্মুক্ত করে।
 
-## Source plugins
+## সোর্স প্লাগিন সমূহ
 
-New nodes are added to Gatsby by "source" plugins. A common one that many Gatsby
-sites use is the [Filesystem source plugin](/packages/gatsby-source-filesystem/)
-which turns files on disk into File nodes.
+Gatsbyতে নতুন নোড "সোর্স" প্লাগইনগুলি দ্বারা যুক্ত করা হয়. একটি প্রচলিত "সোর্স" প্লাগইন যা অনেক গ্যাটসবি
```
Comment (nutboltu) [inline docs/docs/node-interface.md]: গ্যাটসবি -> Gatsby

File: docs/docs/node-interface.md [hunk 3]
```diff
@@ -27,66 +27,66 @@ internal: {
...
+সাইট দ্বারা ব্যবহৃত হয় সেটা হলো [Filesystem source plugin](/packages/gatsby-source-filesystem/)
```
Comment (nutboltu) [inline docs/docs/node-interface.md]: Filesystem source plugin -> ফাইল সিস্টেম সোর্স প্লাগিন

File: docs/docs/node-interface.md [hunk 4]
```diff
@@ -27,66 +27,66 @@ internal: {
...
 [Hacker News](/packages/gatsby-source-hacker-news/)
 
-## Transformer plugins
+## ট্রান্সফরমার প্লাগিন সমূহ
 
-Transformer plugins can also create nodes by transforming source nodes into new
-types of nodes. It is very common when building Gatsby sites to install both
-source plugin(s) and transformer plugins.
+ট্রান্সফর্মার প্লাগইনগুলি সোর্স নোডগুলিকে রূপান্তর করে নতুন ধরণের নোড তৈরি করতে পারে|
+গ্যাটসবি সাইট তৈরি করার সময় সোর্স প্লাগিন সমূহ আর ট্রান্সফরমার প্লাগিন সমূহ
```
Comment (nutboltu) [inline docs/docs/node-interface.md:71]: গ্যাটসবি -> Gatsby

**Review by nutboltu — CHANGES_REQUESTED** [review]
@GeekBoySupreme Sorry for the late response 😞 . I made few minor comments. the Rest of them looks good to me. I think we can merge it whenever you update the PR.

File: docs/docs/node-interface.md (hunk 4 above)
Comment (GeekBoySupreme) [reply docs/docs/node-interface.md:71]: Updated these. Had missed a few

**Comment by GeekBoySupreme** [comment]
@nutboltu no worries. Made the relevant updates

**Review by nutboltu — APPROVED** [review]
awesome work! @GeekBoySupreme

**Pull request merged**